README.md

```diff
@@ -14,38 +16,28 @@ This type of fundraising might allow for more flexibility, accountability, and a
 
-![](./diagrams/out/bonding_curve_buy_flow.png)
+![](./docs/diagrams/out/bonding_curve_buy_flow.png)
 
 - When a user **sells** bonded tokens: The bonded tokens are burned and the seller receives collateral tokens based on the current price.
 
-![](./diagrams/out/bonding_curve_sell_flow.png)
+![](./docs/diagrams/out/bonding_curve_sell_flow.png)
 
 - When the DAO earns **revenue:** the income is split between direct income to the DAO treasury and **dividends** for the bonded token holders.
 
-![](./diagrams/out/bonding_curve_pay_flow.png)
+![](./docs/diagrams/out/bonding_curve_pay_flow.png)
 
 ## Curve Economics
 
 - The specifics of curve design are an area of active research. A common general principle is to reward earlier contributors without inflating the buy price so high that it disincentives later participation.
 
 - Our current curve implementation is based on **bancor formula**, which allows for a wide variety of potential curves with simple parameters.
-  ![](./diagrams/out/bancor_curve_examples.png)
+  ![](./docs/diagrams/out/bancor_curve_examples.png)
 
 [Bancor Whitepaper](https://storage.googleapis.com/website-bancor/2018/04/01ba8253-bancor_protocol_whitepaper_en.pdf)
 
 ## Dividend Tracking
 
-- Scalable dividends purely on-chain is a challenge due to computation and storage costs. We've designed a Merkle-tree based payment pool based on the [Cardstack implementation](https://github.com/cardstack/merkle-tree-payment-pool) that uses Merkle proofs to enable highly scalable dividend distributions with minimal on-chain cost.
-
-- From a user's perspective, the mechanism functions in a conceptually similar way to stock dividends. For each **claim period** (the duration of which can be specified by the DAO), a snapshot of bonded token balances is taken and users can withdraw dividends proportional to their holdings at that time.
-
-- For each claim period, a new merkle root is calculated by a DAO member client side with [our library](https://github.com/statesauce/merkle-tree-payment-pool) [(utilizing merkletreejs)](https://github.com/miguelmota/merkletreejs) and uploaded as a proposal to the DAO. The full information is stored on IPFS. Once the DAO ratifies this proposal, the member who generated the merkle root is compensated for their efforts and token holders are able to withdraw for the period.
-
-![](./diagrams/out/bonding_curve_merkle_flow.png)
-
-- For a discussion of the potential vulnerabilities of this approach and their mitigation, please see the 'Current Limitations' section.
-
-- We have an alternative potential dividend implementation based on the [MiniMe Token](https://github.com/Giveth/minime/blob/master/contracts/MiniMeToken.sol) for a simpler approach that requires significantly more on-chain computation.
+- We allow for scalable, purely on-chain tracking of dividends. [More information here](./docs/DividendToken).
```

Comment:
Should we focus on it being "scalable" . . . what would that mean in this case?

Reply:
It seemed to play well in our previous discussions - the idea is that it can scale to infinite number of users and payments no problem - ethereum speed is the bottleneck.